component/main.jsonnet

```diff
@@ -204,4 +204,6 @@ local namespace_groups = (
       },
     },
   '60_prometheus_rules': alert_rules.rules,
-} + (import 'kibana-host.libsonnet')
+}
++ (import 'kibana-host.libsonnet')
++ (import 'deployment-patch.libsonnet')
```

Comment:
I think we should address scheduling of operator pods which are managed through component openshift4-operators in that component (https://github.com/appuio/component-openshift4-operators) and not in the components which make use of component openshift4-operators.

I'd suggest to simply set the `openshift.io/node-selector: node-role.kubernetes.io/infra=` annotation on the namespaces managed by that component.

Alternatively the component library of component openshift4-operators could support configuring deployment patches, but I don't think that's actually necessary in this case, since we'll always want all operators to run on the infra nodes, if there are any.

Reply:
Looks like I copied that from the official instructions, cf. https://docs.openshift.com/container-platform/4.9/logging/cluster-logging-deploying.html#cluster-logging-deploy-cli_cluster-logging-deploying but I don't see how that's necessary unless some operators need to run on the control plane nodes

Reply:
👍  makes for a way simpler change

Reply:
> +1 makes for a way simpler change

IMO the even bigger upside is that we don't need to copy-paste this change to any component which installs an operator via the openshift4-operators component library.